x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plan/logical/InlineStats.java

```diff
@@ -43,43 +32,33 @@
 /**
  * Enriches the stream of data with the results of running a {@link Aggregate STATS}.
  * <p>
- *     This is a {@link Phased} operation that doesn't have a "native" implementation.
- *     Instead, it's implemented as first running a {@link Aggregate STATS} and then
- *     a {@link Join}.
+ *     Maps to a dedicated Join implementation, InlineJoin, which is a left join between the main relation and the
+ *     underlying aggregate.
  * </p>
  */
-public class InlineStats extends UnaryPlan implements NamedWriteable, Phased, Stats {
```

Comment:
Phased is no longer needed since there are no longer embedded nodes - these are now exposed in the plan, as first-class expressions; it's up to the planner, mapper and session to properly address them.